### ISSUE TITLE
Android app to F-Droid?

### ISSUE THREAD

**Issue by OffifialMITX** [guillaumechereau/goxel#199]
> [F-Droid](https://f-droid.org/) is a […] repository of FOSS apps, along with an Android client to perform installations and updates, and news, reviews and other features covering all things Android and software-freedom related.

In other words, [F-Droid](https://f-droid.org/en/about/) is a catalogue of apps that don't have any proprietary code or dependencies, and it's client is commonly used as a substitute to Google's proprietary Play Store.

It would be nice if you could get this app on it, although it's understandable if you don't want, since you charge for it on the Play Store and F-Droid does not include an option for paid apps

**Comment by OffifialMITX** [comment]
I added this app as an [inclusion request](https://gitlab.com/fdroid/rfp/-/issues/1380).

**Comment by guillaumechereau** [comment]
I can't do that at the moment because part of the mobile code is not open.  Maybe in the future though, why not.

**Comment by fossdd** [comment]
Okay, would great if the Source Code comes avaiable to continue the [inclusion request](https://gitlab.com/fdroid/rfp/-/issues/1380).

**Comment by adrielsand** [comment]
@guillaumechereau Any updates?

**Comment by hendursaga** [comment]
> part of the mobile code is not open

Is that still the case?

**Comment by guillaumechereau** [comment]
Yes it is still the case.  I think it will probably stay that way for the time being since the mobile versions are the only source of income I can make from Goxel so far I and am afraid if I make the code open there would be lot of low effort free clones on the market.
